cli/src/semgrep/core_runner.py

```diff
@@ -852,6 +869,8 @@ def validate_configs(self, configs: Tuple[str, ...]) -> Sequence[SemgrepError]:
                 + list(configs)
             )
 
+            print(f"{' '.join(cmd)}")
```

Comment:
Should this be here?

Reply:
No. It was deleted in the previous incarnation of this PR (the first wave of changes), and I hadn't gotten around to updating this one yet.